### PR TITLE
Fix broken endpoint

### PR DESCRIPTION
**Pull request by gibbsoft**
When using eu-west-2 region (maybe other regions too, but not verified) the logs endpoint was incorrect causing all log forwarding to fail.  The default seemed to be taken from the AZ name, rather than the region name.  I have added a fact and template to fix this.

I also discovered that the default stream name was not properly interpolating because of spaces with the braces.  My second commit correct this.

### CODE REVIEW

**Comment by heitorlessa** [comment]
+1

File: tasks/main.yml [hunk 1]
```diff
@@ -15,6 +15,21 @@
   include: "DebianInstall.yml"
   when: ansible_os_family == "Debian"
 
+- name: "Determine region of the instance"
```
Comment (dharrisio) [inline tasks/main.yml]: I haven't been able to reproduce this issue. If you are still running into issues, can you provide the version of Ansible, and the OS (the AMI if possible). I'm guessing this was an issue with the ec2-facts module, which has been fixed.

File: defaults/main.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ---
 # defaults file for ansible-aws-cloudwatch-logs-agent
 extra_logs: {}
-stream_name: "{ instance_id }"
+stream_name: "{instance_id}"
```
Comment (dharrisio) [inline defaults/main.yml]: I've reproduced this issue, can you submit this as a separate pr?

**Comment by dharrisio** [comment]
@gibbsoft Sorry about the delay here, let me know if you are still running into the region issue.

Thanks!

**Comment by dharrisio** [comment]
The interpolation issue has been merged with #8.
